client/packages/cli/index.js

```diff
@@ -604,18 +672,57 @@ async function init() {
       { name: "Create a new app", value: "create" },
       { name: "Import an existing app", value: "import" },
     ],
-  });
+  }).catch(() => null);
 
   if (action === "create") {
-    await createApp(pkgAndAuthInfo);
-    return;
+    return await promptCreateApp();
   }
 
-  if (action === "import") {
-    const appId = await promptForAppId(pkgAndAuthInfo);
-    if (!appId) return;
+  return await promptImportAppOrCreateApp();
+}
+
+async function handleCreatedApp(
+  { pkgDir, instantModuleName },
+  appId,
+  appTitle,
+) {
+  const schema = await readLocalSchemaFile();
+  const { perms } = await readLocalPermsFile();
+
+  console.log(chalk.green(`Successfully created your Instant app "${appId}"`));
+  console.log(`Please add your app ID to your .env config:`);
+  console.log(chalk.magenta(`INSTANT_APP_ID=${appId}`));
+  console.log(chalk.underline(appDashUrl(appId)));
```

Comment:
Would be nice to use terminalLink here so supported terminals would get a clickable link, e.g. https://github.com/instantdb/instant/blob/main/client/packages/cli/index.js#L832

Reply:
Oo this is great! I will include this in a follow-on PR, where I update every link with `terminalLink`